base.js

```diff
@@ -25,26 +28,65 @@ module.exports.LengthResult = LengthResult;
 module.exports.WriteResult = WriteResult;
 module.exports.ReadResult = ReadResult;
 
-function BufferRW(byteLength, readFrom, writeInto) {
+function BufferRW(poolByteLength, poolReadFrom, poolWriteInto) {
```

Comment:
BufferRW is arguably a public interface, however luckily, ThriftRW and TChannel do not use it directly.